api/src/main/java/org/openmrs/module/queue/model/EncounterStatus.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.queue.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import java.util.Date;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import org.openmrs.BaseOpenmrsMetadata;
+import org.openmrs.Concept;
+import org.openmrs.Encounter;
+import org.openmrs.EncounterType;
+import org.openmrs.Patient;
+import org.openmrs.Visit;
+
+@EqualsAndHashCode(onlyExplicitlyIncluded = true, callSuper = true)
+@NoArgsConstructor
+@Data
+@Entity
+@Table(name = "encounter_status")
+public class EncounterStatus extends BaseOpenmrsMetadata {
```

Comment:
EncounterStatus, like Encounter, is really data rather than metadata (that is to say, we want the semantics of "voided" rather than "retired").

```suggestion
public class EncounterStatus extends BaseChangeableOpenmrsData {
```

Reply:
I guess `BaseChangeableOpenmrsData` is only available in 2.2+

Reply:
I've always treated "retired" & "voided" as the same thing. Now I know, Thanks!